src/init.py

```diff
@@ -41,10 +40,10 @@ def __assign_displays(self):
         self.no_of_displays = len(self.displays)
         self.no_of_connected_dev = self.no_of_displays
 
-        if self.no_of_displays is 1:
+        if self.no_of_displays == 1:
 
```

Comment:
hm, why this change? python3 supports this; and it is more readable.

Reply:
It gave some warning in Python 3.8 that it might be removed in future

Reply:
can you please put the warning here so that I can dig into it?

Reply:
`./init.py:43: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if self.no_of_displays is 1:`

Reply:
@LordAmit I misunderstood the warning. It's probably not going to be completely removed from python, but now is and == have different uses. Check this. https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/

Reply:
Yup; I understand. we better use ==.